lib/repository/forum/ForumDradenMeldingRepository.php

```diff
@@ -112,27 +118,22 @@ public function getAltijdMeldingVoorDraad(ForumDraad $draad) {
 	 * @param string $bericht
 	 */
 	private function stuurMelding($ontvanger, $auteur, $post, $draad, $bericht) {
-		$values = array(
-			'NAAM' => $ontvanger->getNaam('civitas'),
-			'AUTEUR' => $auteur->getNaam('civitas'),
-			'POSTLINK' => $post->getLink(true),
-			'TITEL' => $draad->titel,
-			'TEKST' => str_replace('\r\n', "\n", $post->tekst),
-		);
 
 		// Stel huidig UID in op ontvanger om te voorkomen dat ontvanger privé of andere persoonlijke info te zien krijgt
-		ContainerFacade::getContainer()->get(SuService::class)->overrideUid($ontvanger->uid);
+		$this->suService->alsLid($ontvanger->account, function () use ($ontvanger, $auteur, $post, $draad, $bericht) {
```

Comment:
`$ontvanger->account` kan null zijn als het account van iemand die een draad volgt verwijderd is.

Bij vermelding wordt dit wel afgevangen:
https://github.com/csrdelft/csrdelft.nl/blob/2818035e8cf14edc2b5f6fb20fe5c600c51f8821/lib/repository/forum/ForumDradenMeldingRepository.php#L156